### PR TITLE
Fix unknown stm32 symbols

### PR DESCRIPTION
**Pull request by erwango**
Aim of this commit is to clean STM32 code from unknow Kconfig symbols.

Fixes: #7795

### CODE REVIEW

**Review by galak — CHANGES_REQUESTED** [review]
Looks good, a few typo's in commit messages (spelling errors, etc).

**Comment by erwango** [comment]
CI issue due to tests/crypto/hmac_prng. I expect this will be solved in master

**Review by galak — APPROVED** [review]

File: drivers/clock_control/stm32f3x_ll_clock.c [hunk 1]
```diff
@@ -46,8 +46,13 @@ void config_pll_init(LL_UTILS_PLLInitTypeDef *pllinit)
 	 * ...
 	 * 16 -> LL_RCC_PLLSOURCE_HSE_DIV_16 -> 0x0001000F
 	 */
-	pllinit->PLLDiv = (RCC_CFGR_PLLSRC_HSE_PREDIV |
-					(CONFIG_CLOCK_STM32_PLL_PREDIV1 - 1));
+	/* TODO: Enable this code and matching CONFIG symbol once one of */
```
Comment (nashif) [inline drivers/clock_control/stm32f3x_ll_clock.c]: that comment style is weird,

use this instead...
```
/*
 *
 *
 *
 */
```

Comment (erwango) [reply drivers/clock_control/stm32f3x_ll_clock.c]: Sure. Done.

**Pull request merged**